compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/test/aie_elf_files_gen_test.cpp

```diff
@@ -111,6 +111,7 @@ int main(int argc, char **argv) {
           /*timing=*/false,
           /*tempDir=*/workDir.str().str(),
           /*useChess=*/false,
+          /*useChessForUKernel=*/false,
```

Comment:
Wouldn't this also be `true` ?

Reply:
It doesn't matter because it's not being called, but I explicitly don't want unit tests to depend on chess, so should be false.